bundles/org.eclipse.swt/Eclipse SWT/common/org/eclipse/swt/widgets/TypedListener.java

```diff
@@ -63,11 +63,26 @@ public class TypedListener implements Listener {
  *
  * @noreference This method is not intended to be referenced by clients.
  */
+@Deprecated(forRemoval=true, since="2025-03")
```

Comment:
With https://github.com/eclipse-platform/eclipse.platform.swt/pull/1616, this entire class is now deprecated, so we don't have to deprecate this single constructor separately anymore.